src/openbf.f

```diff
@@ -169,18 +169,19 @@ RECURSIVE SUBROUTINE OPENBF(LUNIT,IO,LUNDX)
 
       IF(IO.EQ.'QUIET') THEN
 c  .... override previous IPRT value (printout indicator)
-         IF(LUNDX.LT.-1)  LUNDX = -1
-         IF(LUNDX.GT. 2)  LUNDX =  2
-         IF(LUNDX.GE.0) THEN
+         IPRTPRV = IPRT
+         IPRT = LUNDX
```

Comment:
Seems like you are adding or changing code, not just addding a test. Is this intentional?

Reply:
If I understand you correctly you've fixed the code so that the segfault cannot occur with bad input to the function - is that correct?

If so, did you also add testing for that?

Reply:
Yes, that's correct, and yes I'm testing for that with the following two lines of intest10.F90 which test these out-of-bounds conditions:

```
  call openbf ( 21, 'QUIET', 3 )
  call openbf ( 21, 'QUIET', -2 )
```

As noted earlier, this is how I found the error in the first place and can now verify that the fix I made is working! ;-)